src/plugins/console/server/lib/spec_definitions/json/overrides/search_application.post_behavioral_analytics_event.json

```diff
@@ -0,0 +1,45 @@
+{
+  "search_application.post_behavioral_analytics_event": {
+    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/master/post-analytics-collection-event.html",
+    "patterns": [
+      "_application/analytics/{collection_name}/event/{event_type}",
```

Comment:
question: "_application/analytics/{collection_name}/event/{event_type}" is this needed now that the click, page_view and search are added?

Reply:
i don't understand the mechanics behind it but if i don't have this then whatever pattern is first isn't suggested as an option 🤷🏻‍♀️

Reply:
From what I understand the use of curly braces would allow any dynamic value to be used for `event_type`, so if there are only 3 possible options, I would recommend deleting the first item. 
I was able to see the 3 options correctly autocompleted with the line removed. 
#### With `{event_type}`
<img width="923" alt="Screenshot 2023-06-02 at 18 50 53" src="https://github.com/elastic/kibana/assets/6585477/7e1dba4c-31f5-4890-88f7-f3ca8c40f5a6">

#### Without `{event_type}`
<img width="883" alt="Screenshot 2023-06-02 at 18 48 44" src="https://github.com/elastic/kibana/assets/6585477/38f2fa3e-6b5c-411d-8411-e68fdd738794">

Reply:
https://github.com/elastic/kibana/pull/158674/commits/96bdb441f848bd559fa56ab8f2bf38690b12ae31